sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -642,7 +658,16 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         frameworks.zip(frameworkAdapters).collect {
           case (tf, Some(adapter)) => (tf, adapter)
         }.toMap
-      }.dependsOn(npmUpdate).value
+      }.dependsOn(npmUpdate).value,
+
+      onComplete := {
```

Comment:
I *think* this needs to be in `override lazy val globalSettings: Seq[Setting[_]]` instead of here. Otherwise it won't have any effect.

You can test that locally with a `println("TEST")` in the lambda below, and make sure that this gets printed. Afterwards, don't forget to remove it when committing.

Reply:
Oh yes, thank you

Reply:
```
sbt:root> test
[info] DummyTest:
[info] - should test stuff
[info] Run completed in 54 milliseconds.
[info] Total number of tests run: 0
[info] Suites: completed 0, aborted 0
[info] Tests: succeeded 0, failed 0, canceled 0, ignored 0, pending 0
[info] No tests were executed.
>>> OnComplete triggered <<<
[success] Total time: 13 s, completed Jan 31, 2018 9:40:27 AM
```